packages/paste-website/src/utils/stickyUtils.ts

```diff
@@ -1,15 +1,13 @@
-import {PSA_ALERT_HEIGHT, PASTE_THEME_ALERT_HEIGHT, SITE_MASTHEAD_HEIGHT} from '../constants';
+import {PSA_ALERT_HEIGHT, SITE_MASTHEAD_HEIGHT} from '../constants';
 
-export const getMainContentComputedOffset = (isPasteTheme: boolean): number => {
-  if (isPasteTheme) {
-    return PSA_ALERT_HEIGHT + PASTE_THEME_ALERT_HEIGHT;
-  }
+export const getMainContentComputedOffset = (): number => {
```

Comment:
Are any of these even computed anymore? They all just return static numbers now, always. Could just be converted to constants

Reply:
You're right. I refactored these.